esrally/mechanic/mechanic.py

```diff
@@ -651,11 +782,13 @@ def load_team(cfg, external):
     else:
         team_path = team.team_path(cfg)
         car = team.load_car(team_path, cfg.opts("mechanic", "car.names"), cfg.opts("mechanic", "car.params"))
-        plugins = team.load_plugins(team_path, cfg.opts("mechanic", "car.plugins"), cfg.opts("mechanic", "plugin.params"))
+        plugins = team.load_plugins(team_path, cfg.opts("mechanic", "car.plugins"),
+                                    cfg.opts("mechanic", "plugin.params"))
     return car, plugins
 
 
-def create(cfg, metrics_store, all_node_ips, cluster_settings=None, sources=False, build=False, distribution=False, external=False,
+def create(cfg, metrics_store, all_node_ips, cluster_settings=None, sources=False, build=False, distribution=False,
+           external=False,
```

Comment:
In general it's hard on me as a reviewer to go through several such unrelated changes on an already large and complex PR so it would be nice if we could move this in the future to dedicated PRs. Also, in this special case we should move the parameter `docker=False` one line up, otherwise we have `external=False` and `docker=False` each on a separate line which looks odd.